tests/integration/internal/k8s/pods.go

```diff
@@ -0,0 +1,67 @@
+package k8s
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/retry"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Same as WaitUntilPodsAvailableE, but terminates the test instead of returning an error.
+func WaitUntilPodsAvailable(
+	t *testing.T,
+	options *k8s.KubectlOptions,
+	filters v1.ListOptions,
+	minPods int,
+	sleepDuration time.Duration,
+	sleepBetweenRetries time.Duration,
+) {
+	require.NoError(t, WaitUntilPodsAvailableE(t, options, filters, minPods, sleepDuration, sleepBetweenRetries))
+}
+
+// WaitUntilPodsAvailableE waits until pods satisfying the provided filters are all available.
+// Availability is defined via terrastruct's k8s.IsPodAvailable function, which requires the Pod
+// to be Running and all of its containers to be started and ready.
```

Comment:
> ...to be started and ready

It's not technically true, right? We're not waiting for readiness probes here, are we?

Reply:
We're waiting for containers to be Ready, which does imply probes afaik:
https://github.com/gruntwork-io/terratest/blob/3d69d62aab234c3eb39adcac1aaff09dd8840697/modules/k8s/pod.go#L140